compiler/rustc_codegen_llvm/src/llvm/mod.rs

```diff
@@ -172,6 +172,12 @@ pub fn SetUnnamedAddress(global: &Value, unnamed: UnnamedAddr) {
     }
 }
 
+pub fn SetVisibility(global: &Value, visibility: Visibility) {
+    unsafe {
+        LLVMRustSetVisibility(global, visibility);
+    }
+}
```

Comment:
There’s already a `set_visibility` function. I would argue that we should largely try to follow the snake_case here and the `SetUnnamedAddress` et al are the odd ones out.